src/assets/scss/application.scss

```diff
@@ -290,4 +290,59 @@ $nav-active-border-thickness: 5px;
   display: block;
   margin: govuk-spacing(2) 0;
   @include govuk-font($size: 48, $weight: bold);
-}
\ No newline at end of file
+}
+
+.language-select {
+  margin: govuk-spacing(2) 0;
+
+  &::after {
+    content: "";
+    display: block;
+    clear: both;
+  }
+
+  .language-select__list {
+    border-bottom: 1px solid $govuk-border-colour;
+    margin-bottom: 0;
+    float: right;
+    text-align: right;
+    @include govuk-media-query($until: tablet) {
+      float: none;
+      text-align: left;
+    }
+  }
+
+  .language-select__link {
+    @include govuk-font($size: 16);
+    padding: 0.3125em;
+
+    &:visited {
+      color: $govuk-link-colour;
+
+      &:hover {
+        color: $govuk-link-hover-colour;
+      }
+
+      &:focus {
+        color: $govuk-focus-text-colour;
+      }
+    }
+  }
+
+  .language-select__list-item {
+    display: inline-block;
+    margin-right: 0;
+    border-bottom: 0;
+  }
+
+  .language-select__list-item [aria-current] {
+    padding: 0.3125em;
```

Comment:
Interested in this `0.3125em` value! "Seems a little awkward. 
Is this something that we cannot achieve with standard `govuk-spacing`?
Is there a reason this component should be special and should not use standard spacing, like the [GOV.UK version](https://components.publishing.service.gov.uk/component-guide/translation_nav) does?

Reply:
Totally agree, this is a weird one. This is the value from the prototype. Not sure why they have spacing though but I have just changed it with the govuk-spacing equivalent.